src/components/IconButton/IconButton.jsx

```diff
@@ -49,7 +49,6 @@ const IconButton = forwardRef(
 
     const overrideStyle = useMemo(() => {
       let style = {
-        display: "inline-flex",
```

Comment:
defined inside button and until now was not active because include typo in inline-flex